src/card.css

```
@@ -0,0 +1,118 @@
+:root {
+  --color-white: #fff;
+  --color-gray: #f3f3f3;
+  --color-main-accent: #060b35;
+  --color-secondary: #616070;
+  --color-blue-accent: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  padding: 32px 16px 16px;
+  max-width: 200px;
```

we always want it to be 200px
```suggestion
  width: 200px;
```